wiki/test-case-triggered-event-aggregation.md

```diff
@@ -74,8 +74,8 @@ the following object’s id is selected:
 
     ["6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43"]
 
-But there is no object with such id in the database and aggregatedObject
-returns empty. Event is added to the waitlist. There it waits until the object
+But there is no object with such id in the database and aggregated object
+returns empty. Event is added to wait list. There it waits until the object
```

Comment:
I guess the merge went wrong here because I believe you changed it to _the waitlist_ ?

Reply:
Done